egs/chime6/s5_track2/local/decode.sh

```diff
@@ -8,13 +8,15 @@
 # Apache 2.0
 
 # Begin configuration section.
-nj=8
+nj=10
 decode_nj=10
```

Comment:
Does `decode_nj=10` works for you? In CHiME-6 `S09_U05` (also in CHiME-5) and `S01_U03` (chime6-synchronisation has problems) are missing.

Later in this file the loop `for mictype in u01 u02 u03 u04 u06; do` ignores `U05` for DEV and EVAL. In DEV we can then use 10 jobs. Since in EVAL `S01_U03` is missing I got an error, because I was only allowed to use 9 jobs with `run.pl`.

Reply:
Good catch. I'll change that to be equal to the number of lines in utt2spk.